main.go

```diff
@@ -154,6 +153,14 @@ func runCopy(db *redshift.Redshift, inputConf s3filepath.S3File, inputTable reds
 	if err := tx.Commit(); err != nil {
 		return fmt.Errorf("err committing transaction: %s", err)
 	}
+
+	if truncate {
+		// If we've truncated the table we should run vacuum and analyze to sort the
+		// data, remove the deleted rows, and update the statistics.
+		if err := db.VacuumAnalyze(inputConf.Schema, inputTable.Name); err != nil {
```

Comment:
Sounds like a plan. I'd rather start small and work bigger - then we know which changes impacted performance.